occ.py

```diff
@@ -150,9 +151,15 @@ async def parse_commit(payload, config):
                         break
 
 
+def ocCli():
+    parser = argparse.ArgumentParser(description='infrastructure-occ')
+    parser.add_argument('--config-file', type=str, help='YAML configuration file.')
+    return parser.parse_args()
+
 async def main():
-    print("Loading occ.yaml")
-    cfg = yaml.safe_load(open('occ.yaml'))
+    args = ocCli()
+    print("Loading %s" % args.config_file)
+    cfg = yaml.safe_load(open(args.config_file))
```

Comment:
This appears to be a breaking change, in that the existing code expects no command line arguments and defaults to occ.yaml. Your proposed change would cause existing deployments to fail unless their calling conventions were also updated. It should default to opening occ.yaml if no --config-file argument is provided.

Reply:
Ok, looks like the solution is to add ```default="occ.yaml"``` to ```parser.add_argument()``` as following:

```

def ocCli():
    parser = argparse.ArgumentParser(description='infrastructure-occ')
    parser.add_argument('--config-file', type=str, default="occ.yaml", help='YAML configuration file. Default is: ./occ.yaml')
    return parser.parse_args()

```

I also extended the ```help``` of OCC as per requested updates.

If I try other conditions  than adding ```default="occ.yaml"``` , I get strange behaviors when calling ```occ.py -h```.

---

Result without specifying ```--config-file``` option:

```
root@ubuntu:/usr/src/iocc/infrastructure-occ# ./v/bin/python occ.py
occ.yaml loaded.
Listening to pyPubSub stream at https://pubsub.apache.org:2070/commit

```

Result specifying ```--config-file``` option:

```
root@ubuntu:/usr/src/iocc/infrastructure-occ# ./v/bin/python occ.py --config-file /etc/occ/occ.yaml
/etc/occ/occ.yaml loaded.
Listening to pyPubSub stream at https://pubsub.apache.org:2070/commit
 
```

Reply:
Changes applied.